xarray/tests/test_convert.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import xarray as xr
+import iris
+from iris.coords import DimCoord
+from iris.cube import Cube
+
+
+def test_from_iris_no_var_name():
+    latitude = DimCoord(np.linspace(-90, 90, 4),
+                        standard_name='latitude',
+                        units='degrees')
+    cube = Cube(np.zeros((4,), np.float32),
+                dim_coords_and_dims=[(latitude, 0)])
+    xr.DataArray.from_iris(cube)
+
+
+def test_to_iris_non_numeric_coord():
+    data = np.random.rand(3)
+    locs = ['IA', 'IL', 'IN']
+    da = xr.DataArray(data, coords=[locs], dims=['space'])
+    xr.DataArray.to_iris(da)
```

Comment:
Here you could check that the Iris object is what you expect. I'm not sure if Iris has a function for testing if cubes are identical -- maybe @duncanwp or @pelson knows?

Reply:
You can just use the equallity cube_a == cube_b. This will check the metadata, coordinates and data (within a tolerance) are identical.